gradio/cli/commands/components/docs.py

```diff
@@ -0,0 +1,101 @@
+import importlib
+from pathlib import Path
+from typing import Optional
+
+import requests
+import toml
```

Comment:
toml is only included in 3.10. We need to use `tomlkit` here for older versions. It's already installed and AKAIK `toml` is identical to `tomlkit`.